### PR TITLE
Pull request: Update Japanese Translation on 29 Aug, 2021

### PR DESCRIPTION
**Pull request by Proxyer**
resend pull request.

### CODE REVIEW

File: Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-
-</LanguageData>
\ No newline at end of file
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LanguageData>
+
+	<!-- ========== Royalty DLC-exclusive content ========== -->
+
+	<!-- EN: MedPod Lux -->
+	<MedPodLux.label>治療ポッド・高級</MedPodLux.label>
```
Comment (sumghai) [inline Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml:7]: Um, you still haven't changed this back to Lux.

File: Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+
+	<!-- EN: An advanced hospital bed that uses accelerated cellular regeneration to treat nearly all kinds of injuries and diseases, including regenerating missing organs and limbs.\n\nThis is an luxury model with gold-inlaid fittings and a gel-infused memory foam mattress, capable of faster diagnosis and treatment but requires higher power consumption. -->
+	<MedPodLux.description>欠損した臓器や手足の再生など、ほぼすべての負傷や疾患の治療を加速する細胞再生技術を利用した高度な医療ベッドです。\nゴールドを使った豪華な装飾とジェルを包んだ形状記憶フォームマットレスを使用した贅沢な形式で、より高速に診断と治療を行えますが、消費電力が更に増えます。</MedPodLux.description>
+
+	<!-- EN: MedPod (blueprint)-->
+	<Blueprint_MedPodLux.label>治療ポッド・高級(設計図)</Blueprint_MedPodLux.label>
```
Comment (sumghai) [inline Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml:13]: Ditto.

File: Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+
+	<!-- EN: MedPod (blueprint)-->
+	<Blueprint_Install_MedPodLux.label>治療ポッド・高級(移動先)</Blueprint_Install_MedPodLux.label>
```
Comment (sumghai) [inline Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml:16]: Ditto.

File: Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+
+	<!-- EN: MedPod (building)-->
+	<Frame_MedPodLux.label>治療ポッド・高級(建設中)</Frame_MedPodLux.label>
```
Comment (sumghai) [inline Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml:19]: Ditto.

**Comment by Proxyer** [comment]
- At least 1 approving review is required by reviewers with write access. 
- Merging can be performed automatically with 1 approving review.
- This is looks like a approve problem.

**Review by sumghai — CHANGES_REQUESTED** [review]
Um, you still haven't changed 高級 back to Lux.

Also, if you need to fix a Pull Request, you **don't** need to close it and start over - just push new commits to the existing PR.

**Comment by Proxyer** [comment]
>Um, you still haven't changed 高級 back to Lux.
>Also, if you need to fix a Pull Request, you don't need to close it and start over - just push new commits to the existing PR.

Probably the Japanese translation of MedPod Lux is not the cause of the error.
It is actually displayed without any problem on the in-game screen. It could also be built.
https://steamcommunity.com/sharedfiles/filedetails/?id=2588210540

**Comment by Proxyer** [comment]
I went back to Lux, but I still get the error. It was not the cause of the problem.

File: Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml (hunk 1 above)
Comment (sumghai) [reply Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml]: Fixed by @Proxyer in 6bfccf0

File: Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml (hunk 2 above)
Comment (sumghai) [reply Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml]: Fixed by @Proxyer in 6bfccf0

File: Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml (hunk 3 above)
Comment (sumghai) [reply Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml]: Fixed by @Proxyer in 6bfccf0

File: Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml (hunk 4 above)
Comment (sumghai) [reply Royalty/Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_DLC_MedPod.xml]: Fixed by @Proxyer in 6bfccf0

**Comment by sumghai** [comment]
> 
> 
>     * At least 1 approving review is required by reviewers with write access.
> 
>     * Merging can be performed automatically with 1 approving review.
> 
>     * This is looks like a approve problem.

This is because (at that time) I hadn't approved the changes yet. Now that the MedPod Lux name has been fixed, I will approve it now.

**Review by sumghai — APPROVED** [review]

**Comment by sumghai** [comment]
@Proxyer - Just a heads-up that since translation fixes usually aren't mission-critical, I'll wait until the next update before officially releasing these changes to the public.

**Comment by Proxyer** [comment]
I undersand. Thank you for your support.

**Pull request merged**